pkgs/applications/editors/vim/plugins/utils/nvim-treesitter/update.py

```diff
@@ -13,7 +13,10 @@
 log = logging.getLogger("vim-updater")
 
 NURR_JSON_URL = (
-    "https://raw.githubusercontent.com/nvim-neorocks/nurr/main/tree-sitter-parsers.json"
+    "https://raw.githubusercontent.com/lumen-oss/nurr/main/tree-sitter-parsers.json"
```

Comment:
Does it pose any concern that this could be slightly out of sync with the parsers specified in the `nvim-treesitter` repo? As per their README, "This plugin is only guaranteed to work with specific versions of language parsers (as specified in the parser.lua table)"

Appears there is some CI in place to keep the NURR json up to date with nvim-treesitter parsers.lua (maybe @mrcjkb can speak to this), but from my angle it seems it'd be safest to have a [versioned URL](https://raw.githubusercontent.com/lumen-oss/nurr/00725adfd544a1c3fd3b25a5b85a1a03ec6f1fb3/tree-sitter-parsers.json) here such that the rev from NURR's `nvim-treesitter.lock` matches the nvim-treesitter rev we're building.